mclagsyncd/mclaglink.cpp

```diff
@@ -401,274 +575,1272 @@ void MclagLink::setFdbEntry(char *msg, int msg_len)
+                    {
+                        attrBmap = (attrBmap | MCLAG_CFG_ATTR_SRC_ADDR);
+                        memcpy(cfg_info.local_ip, domainData.source_ip.c_str(), INET_ADDRSTRLEN);
+                    }
+                }
+                if (fvField(i) == "peer_ip")
+                {
+                    domainData.peer_ip = fvValue(i); 
+                    if(!entryExists)
+                    {
+                        attrBmap = (attrBmap | MCLAG_CFG_ATTR_PEER_ADDR);
+                        memcpy(cfg_info.peer_ip, domainData.peer_ip.c_str(), INET_ADDRSTRLEN);
+                    }
+                }
+                if (fvField(i) == "peer_link")
+                {
+                    domainData.peer_link = fvValue(i); 
+                    if(!entryExists)
+                    {
+                        attrBmap = (attrBmap | MCLAG_CFG_ATTR_PEER_LINK);
+                        memcpy(cfg_info.peer_ifname, domainData.peer_link.c_str(), MAX_L_PORT_NAME);
+                    }
+                }
+                if (fvField(i) == "keepalive_interval")
+                {
+                    if (fvValue(i).empty())
+                    {
+                        domainData.keepalive_interval = -1;
+                    }
+                    else
+                    {
+                        domainData.keepalive_interval = stoi(fvValue(i).c_str()); 
+                    }
+                    if(!entryExists)
+                    {
+                        attrBmap = (attrBmap | MCLAG_CFG_ATTR_KEEPALIVE_INTERVAL);
+                        cfg_info.keepalive_time = domainData.keepalive_interval;
+                    }
+                }
+                if (fvField(i) == "session_timeout")
+                {
+                    if (fvValue(i).empty())
+                    {
+                        domainData.session_timeout = -1;
+                    }
+                    else
+                    {
+                        domainData.session_timeout = stoi(fvValue(i).c_str()); 
+                    }
+                    if(!entryExists)
+                    {
+                        attrBmap = (attrBmap | MCLAG_CFG_ATTR_SESSION_TIMEOUT);
+                        cfg_info.session_timeout = domainData.session_timeout;
+                    }
+                }
+            }
+
+
+            //If entry present send only the diff
+            if (entryExists)
+            {
+                if( (it->second.source_ip.compare(domainData.source_ip)) != 0)
+                {
+                    attrBmap = (attrBmap | MCLAG_CFG_ATTR_SRC_ADDR);
+                    memcpy(cfg_info.local_ip, domainData.source_ip.c_str(), INET_ADDRSTRLEN);
+                    if (domainData.source_ip.empty())
+                    {
+                        attrDelBmap = attrDelBmap | MCLAG_CFG_ATTR_SRC_ADDR;
+                    }
+                }
+                if( (it->second.peer_ip.compare(domainData.peer_ip)) != 0)
```

Comment:
Is this space after ( needed? If not, please remove it from here and from the similar places below.